tests/manager/manager-debug-002.phpt

```diff
@@ -14,7 +14,6 @@ $bulk = new MongoDB\Driver\BulkWrite();
 $bulk->insert(array('_id' => 1, 'x' => 1));
 $result = $manager->executeBulkWrite(NS, $bulk);
 
-ini_set("mongodb.debug", "off");
```

Comment:
Best I can tell, the request reverts to the original INI setting (`mongodb.debug=stderr` from the ``--INI--`` block for this test). Disabling debugging inside the request here does not prevent further output from cleaning up the persistent clients during MSHUTDOWN.